ui/app/components/pending-tx.js

```diff
@@ -298,7 +310,7 @@ PendingTx.prototype.render = function () {
             type: 'submit',
             value: 'ACCEPT',
             style: { marginLeft: '10px' },
-            disabled: insufficientBalance || !this.state.valid,
+            disabled: insufficientBalance || !this.state.valid || !isValidAddress,
```

Comment:
im suspicious of `setState` being async and leaving a window where validation hasnt caught up

but not sure how possible that is